websites/A/Animeflix/presence.ts

```diff
@@ -0,0 +1,62 @@
+const presence = new Presence({
+	clientId: "1039178085922250873",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/4hiyxuW.png",
+			buttons: [
+				{
+					label: "Visit Website",
+					url: "https://animeflix.live/",
+				},
+				{
+					label: "Join Discord",
+					url: "https://discord.gg/SKcb2C7HjH",
+				},
```

Comment:
Buttons that lead to home pages are not allowed.

I'm also not sure if buttons to Discord servers are allowed.